modules/ROOT/pages/sysconfig-configure-kdump.adoc

```diff
@@ -0,0 +1,51 @@
+:experimental:
+= Enabling kdump
+
+== Introduction
+kdump is a service that creates crash dumps when there is a kernel crash. It uses kexec to boot into a secondary kernel (known as a capture kernel), then exports the contents of the kernel's memory (known as a crash dump or vmcore) to the filesystem. The contents of vmcore can then be analyzed to root cause the kernel crash.
+
+kdump dump requires two things to be configured--a kernel argument to reserve memory for the crash kernel and enabling of the kdump systemd service.
+
+== Configure kdump on FCOS through Ignition
+
+.Example kdump configuration
+[source,yaml]
+----
+variant: fcos
+version: 1.4.0
+kernel_arguments:
+  should_exist:
+  - 'crashkernel=300M'
+systemd:
+  units:
+    - name: enable-kdump.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Enable kdump
+        # Run before Zincati as we will reboot
+        Before=zincati.service
+        # We run after `systemd-machine-id-commit.service` to ensure that
+        # `ConditionFirstBoot=true` services won't rerun on the next boot.
+        After=systemd-machine-id-commit.service
+        ConditionPathExists=!/var/lib/%N.stamp
+
+        [Service]
+        Type=oneshot
+        RemainAfterExit=yes
+        ExecStart=/bin/systemctl enable kdump.service
+        ExecStart=/bin/touch /var/lib/%N.stamp
+        ExecStart=/bin/systemctl reboot
+
+        [Install]
+        WantedBy=multi-user.target
+----
+
+[NOTE]
```

Comment:
might be nice to mention somewhere in here the info from the other page about `/etc/kdump.conf`